wire-ios/.gitignore

```diff
@@ -26,4 +26,3 @@ Scripts/**/.build
 /fastlane/report.xml
 /test
 /fastlane/README.md
-/Configuration/
```

Comment:
I would move the public configuration outside of wire-ios/Configuration maybe at the root level and setup just symlink to it.

Reply:
Why?